README.md

```diff
@@ -1476,6 +1476,28 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='generic-extensions'></a>(<a href='#generic-extensions'>link</a>) When extending generic types, prefer using generic bracket syntax (or sugared syntax for applicable standard library types) instead of generic type constraints. [![SwiftFormat: genericExtensions](https://img.shields.io/badge/SwiftFormat-genericExtensions-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#genericExtensions)
```

Comment:
I think it would be helpful to explain the difference between "generic bracket syntax" and "sugared syntax", possibly with examples. This could even be done in the code snippets by highlighting the different options.

Reply:
How about this?

```suggestion
* <a id='generic-extensions'></a>(<a href='#generic-extensions'>link</a>) When extending generic types, prefer using generic bracket syntax (`extension Array<Planet>`), or sugared syntax for applicable standard library types (`extension [Planet]`), instead of generic type constraints. [![SwiftFormat: genericExtensions](https://img.shields.io/badge/SwiftFormat-genericExtensions-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#genericExtensions)
```